packages/react-examples/src/react/List/List.Basic.Example.tsx

```diff
@@ -77,7 +77,7 @@ const onRenderCell = (item: IExampleItem, index: number | undefined): JSX.Elemen
 };
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29748&commitId=45e7d375418ae4736bc7d39dd03487cbb2c3c2b5&env=prod&clientType=FLUENTUI)


  <details><summary><b>DetailsList</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DetailsList.Checkbox Visible Always - RTL.click.chromium.png|3353| Changed|
|DetailsList.Checkbox Visible Always - RTL.default.chromium.png|1969| Changed|
|DetailsList.Checkbox Visible Always - RTL.hover.chromium.png|2498| Changed|

</details>


  <details><summary><b>GroupedList</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|GroupedList.Root - RTL.hover.chromium.png|3259| Changed|
|GroupedList.Root - RTL.default.chromium.png|2623| Changed|
|GroupedList.Root - RTL.click.chromium.png|64994| Changed|

</details>


  <details><summary><b>List</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|List.Root - RTL.default.chromium.png|5831| Changed|

</details>


  <details><summary><b>TilesList</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TilesList.Basic - RTL.default.chromium.png|145680| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
Changes are flakiness. Icons are not loading for the affected test cases.